gcs/gcs.go

```diff
@@ -266,3 +266,57 @@ Note: for some providers, UploadFolder requires that a default bucket be set in
 func (g *GoogleCloudStorage) UploadFolder(foldFace interface{}) ([]*types.UploadedFile, error) {
 	return nil, nil
 }
+
+/*
+DeleteFile deletes a file from a bucket on Google Cloud Storage and returns an error if one occurs.
+
+Note: DeleteFile requires that a default bucket be set in bifrost.BridgeConfig.
+*/
```

Comment:
/*
DeleteFile deletes a file from Google Cloud Storage and returns an error if one occurs.

Note: DeleteFile requires that a default bucket be set in bifrost.BridgeConfig.
*/

Reply:
All changes fixed @opensaucerer